cmd/helm/dependency.go

```diff
@@ -120,3 +121,16 @@ func newDependencyListCmd(out io.Writer) *cobra.Command {
 	f.UintVar(&client.ColumnWidth, "max-col-width", 80, "maximum column width for output table")
 	return cmd
 }
+
+func addDependencySubcommandFlags(f *pflag.FlagSet, client *action.Dependency) {
+	f.BoolVar(&client.Verify, "verify", false, "verify the packages against signatures")
+	f.StringVar(&client.Keyring, "keyring", defaultKeyring(), "keyring containing public keys")
+	f.BoolVar(&client.SkipRefresh, "skip-refresh", false, "do not refresh the local repository cache")
+	f.StringVar(&client.Username, "username", "", "chart repository username where to locate the requested chart")
+	f.StringVar(&client.Password, "password", "", "chart repository password where to locate the requested chart")
```

Comment:
what do you think about adding `--password-stdin` like docker does, for the same reason (setting password in a CLI command is insecure, especially in CI where many folks use Helm commands).

I know password opt is used already, so this may be a bigger issue outside the scope of your PR. May be good to approach all these consistently - eg, we have `passwordFromStdinOpt` only for repo add and registry login, etc.

Reply:
I think I can follow-up with another PR to limit the number of changes in this PR